src/components/app/app.scss

```diff
@@ -0,0 +1,10 @@
+.app {
+  min-height: 100vh;
+  &__container {
+    width: 100%;
+    &-content {
```

Comment:
please do not do nesting this way --- it is enough to write `.app__container` and `.app__content`